### PR TITLE
Run update-ca-trust

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

**Comment by dmage** [comment]
Ready for review.

File: images/dockerregistry/Dockerfile [hunk 1]
```diff
@@ -17,10 +17,13 @@ LABEL io.k8s.display-name="OpenShift Container Platform Image Registry" \
...
+
 # The registry doesn't require a root user.
 USER 1001
 EXPOSE 5000
 VOLUME /registry
 ENV REGISTRY_CONFIGURATION_PATH=/config.yml
 
-CMD /usr/bin/dockerregistry ${REGISTRY_CONFIGURATION_PATH}
+ENTRYPOINT ["sh", "-c", "update-ca-trust && \"$@\"", "arg0"]
+CMD ["/usr/bin/dockerregistry"]
```
Comment (bparees) [inline images/dockerregistry/Dockerfile]: need to update dockerfile.rhel7 also?

Comment (dmage) [reply images/dockerregistry/Dockerfile]: I found 2 more Dockerfiles, all are updated.

**Pull request merged**